### PR TITLE
Allow configurable acceptance tests image

### PR DESCRIPTION
**Pull request by liamtoozer**
# Motivation and Context

Acceptance tests are currently set to run using the CI image each time the job is triggered. It would be useful to run acceptance tests from the pipeline against a custom image, especially now that we can configure our own dev pipelines.  

# What has changed

* Updated README to reflect recent var changes
* Updated pipeline files to allow configurable acceptance test images

# How to test?

* In the pipeline file, add the following to `acceptance-tests-repo` resource:
```yaml
branch: allow-configurable-image-name
```
* fly your own pipeline, ensuring that you set an acceptance tests image in your secrets file

# Links

https://trello.com/c/yYcJCaRz/761-make-acceptance-tests-image-configurable-in-pipeline

### CODE REVIEW

**Comment by liamtoozer** [comment]
[Make acceptance tests image configurable in pipeline](https://trello.com/c/yYcJCaRz/761-make-acceptance-tests-image-configurable-in-pipeline)

File: example-secrets/ci-kubernetes-secrets.yml.example [hunk 1]
```diff
@@ -4,4 +4,5 @@ gcp-project-name:
 gcp-service-account-json:
 github-private-key:
 kubernetes-cluster-name:
-terraform-branch: master
\ No newline at end of file
+terraform-branch: master
+acceptance-tests-image:
```
Comment (jcox-dev) [inline example-secrets/ci-kubernetes-secrets.yml.example]:
```suggestion
acceptance-tests-image: eu.gcr.io/census-rm-ci/rm/census-rm-acceptance-tests:latest
```

**Review by jcox-dev — COMMENTED** [review]
To test these changes you will need to add: 

```yaml
branch: allow-configurable-image-name
```

To the acceptance-tests-repo resource.

**Comment by liamtoozer** [comment]
Good point @jcox-dev, just updated the PR bits with that.

**Review by jcox-dev — APPROVED** [review]

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

**Pull request merged**